src/macho.rs

```diff
@@ -358,6 +518,110 @@ pub struct DyldCacheImageInfo<E: Endian> {
     pub pad: U32<E>,
 }
 
+/// Corresponds to struct dyld_cache_slide_pointer5 from dyld_cache_format.h.
+#[derive(Debug, Clone, Copy)]
+pub enum DyldCacheSlidePointer5 {
```

Comment:
I think this would be better as simply `DyldCacheSlidePointer5(u64)`. No need for `DyldChainedPtrArm64eSharedCacheRebase` or `DyldChainedPtrArm64eSharedCacheAuthRebase`.